tests/integration/targets/init.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC2155,SC2086
 
-BASE_DIR=$(dirname "${BASH_SOURCE[0]}")
-export ANSIBLE_ROLES_PATH=${BASE_DIR}
+export VALIDATED_COLLECTIONS_PATH="/home/kni/actions-runner/eco-vcenter/cloud.vmware_ops/cloud.vmware_ops/ansible_collections"
+export ALL_COLLECTIONS_PATH="$HOME/.ansible/collections/ansible_collections"
```

Comment:
I think we should remove this from the collections path. Other tests or processes could install collections here and cause unexpected results in our tests. I ran into something similar on the github hosted runners, since they come with ansible installed at the system level

Reply:
the issue is that if I left it as it is, the default paths are incorrect, not sure why

Reply:
OK fixed! I just needed to add:
```
        env:
          ANSIBLE_COLLECTIONS_PATH: "${{ github.workspace }}"
```
as you did in your PR. thank you :)